stock_picking_completion/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Stock Picking Completion",
+    "summary": "Compute the stock.picking completion",
+    "version": "14.0.1.0.0",
+    "category": "Inventory",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "mmequignon, Camptocamp SA, Odoo Community Association (OCA)",
```

Comment:
I don't think you have to put your name in the `author` key, authorship belongs to the company (in case of software edition, at least in France).
```suggestion
    "author": "Camptocamp SA, Odoo Community Association (OCA)",
```